### DISCUSSION TITLE
Cloud storage.

### DISCUSSION THREAD

**Discussion by Michaelkire** [night/betterttv#5853] [Ideas]
It would be nice, if you could store the settings data somewhere, so when i login somewhere, i get the settings back.

**Comment by night** [comment]
This is already planned with no ETA.

**Comment by breon** [comment]
Sorry for the duplicate. There isn't a great search for issues.

**Comment by tylervick** [comment]
Would it be possible just to enable the storage permission in the packaged Chrome extension? That way we can test for chrome.storage and use the storage.sync API.

I'd love to see this feature, I've had to set my preferences on so many different machines and it gets tedious.

**Comment by EhsanKia** [comment]
That could be a nice and easy solution. The `chrome.storage.sync` API allows up to 100kb of data to be stored, which is synced with Chrome sync. The settings json is definitely smaller than that, we can simply put it in there and everything else should be pretty automatic. We can also put aliases and other stuff too in there.

**Comment by breon** [comment]
I'd love to see this. Would it also store the whitelist/blacklist? Is that part of the settings json file?

**Comment by tylervick** [comment]
Briefly looking over, this shouldn't be too difficult to add. The blocking part is modifying Chrome's manifest to add the storage permission - is the extension packaging handled separately by you, @night?

**Comment by Trigger911** [comment]
I would love to see this feature too!

**Comment by dclstn** [comment]
After 11 long years, we did it.

You can backup your settings to the cloud.
